packages/pds/src/services/moderation/index.ts

```diff
@@ -420,9 +418,9 @@ export class ModerationService {
       .updateTable('moderation_action')
       .where('id', '=', id)
       .set({
-        reversedAt: createdAt.toISOString(),
-        reversedBy: createdBy,
-        reversedReason: reason,
+        // reversedAt: createdAt.toISOString(),
+        // reversedBy: createdBy,
+        // reversedReason: reason,
```

Comment:
What's the fate of these lines?  Ignore me if you already have this tracked/planned 😏

Reply:
I was just trying to shut the linter up here. this whole thing is gonna go away since we won't be updating any existing event but rather add a new `#revert` event in the stream negating the effects of the event being reverted.